test/autoinst_clone_test.rb

```diff
@@ -0,0 +1,219 @@
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require_relative "test_helper"
+
+Yast.import "AutoinstClone"
+
+describe Yast::AutoinstClone do
+  subject { Yast::AutoinstClone }
+
+  let(:module_map) do
+    {
+
+      "add-on"     => {
+        "Name"                         => "YaST Add-On Products",
+        "Icon"                         => "yast-addon",
+        "X-SuSE-YaST-AutoInst"         => "configure",
+        "X-SuSE-YaST-AutoInstResource" => "add-on",
+        "X-SuSE-YaST-AutoInstClonable" => "true",
+        "X-SuSE-YaST-Group"            => "Software",
+        "X-SuSE-DocTeamID"             => "ycc_org.opensuse.yast.AddOn",
+        "X-SuSE-YaST-AutoInstClient"   => "add-on_auto"
+      },
+      "general"    => {
+        "Name"                         => "General Options",
+        "Icon"                         => "yast-general",
+        "X-SuSE-YaST-AutoInst"         => "configure",
+        "X-SuSE-YaST-AutoInstResource" => "general",
+        "X-SuSE-YaST-Group"            => "System",
+        "X-SuSE-YaST-AutoInstClient"   => "general_auto"
+      },
+      "report"     => {
+        "Name"                         => "Report & Logging",
+        "Icon"                         => "yast-report",
+        "X-SuSE-YaST-AutoInst"         => "configure",
+        "X-SuSE-YaST-AutoInstResource" => "report",
+        "X-SuSE-YaST-Group"            => "System",
+        "X-SuSE-YaST-AutoInstClient"   => "report_auto"
+      },
+      "bootloader" => {
+        "Name"                         => "YaST Boot Loader",
+        "Icon"                         => "yast-bootloader",
+        "X-SuSE-YaST-AutoInst"         => "configure",
+        "X-SuSE-YaST-AutoInstResource" => "bootloader",
+        "X-SuSE-YaST-AutoInstClonable" => "true",
+        "X-SuSE-YaST-Group"            => "System",
+        "X-SuSE-DocTeamID"             => "ycc_org.opensuse.yast.Bootloader",
+        "X-SuSE-YaST-AutoInstClient"   => "bootloader_auto"
+      }
+    }
+  end
+
+  let(:probed_devicegraph) do
+    instance_double(Y2Storage::Devicegraph, multipaths: multipaths)
```

Comment:
well, can you use just some real storage device graphs?

Reply:
Yes, why not? :)